docs/build_arm.md

```diff
@@ -1,43 +1,93 @@
-# Building Larq Compute Engine for ARM-based boards
+# Building Larq Compute Engine for ARM-based systems
 This page descibes how to build a Larq Compute Engine (LCE) inference binary
-for 32-bit, as well as 64-bit ARM-based boards.
-[Bazel](https://bazel.build/) is the primary build
-system for LCE. However, since in some occasions Make build system is a more
-convenient build solution, we also provide scripts to build LCE with Make.
+for 32-bit, as well as 64-bit ARM-based systems. You can either build LCE
+natively on your ARM device or cross-compile it from a non-ARM machine.
+[Bazel](https://bazel.build/) is the primary build system for LCE, and it can
+be used to cross-compile a binary for ARM-based systems using a non-ARM host.
+To natively build on an ARM system itself we provide a script that uses the
+Makefile build system, because Bazel it is currently not natively available on
```

Comment:
```suggestion
... because Bazel it is currently not natively available on
```
I would remove this part bc its possible to install bazel on ARM its just very cumbersome.